src/tigerbeetle/main.zig

```diff
@@ -163,10 +163,12 @@ const Command = struct {
     }
 
     pub fn start(arena: *std.heap.ArenaAllocator, args: *const cli.Command.Start) !void {
```

Comment:
Might change `ArenaAllocator` just for `Allocator` here? We needed arena to print stats, but we now print stats without arena. And I think we can drop "in N regions", as that's about the impl detail of the arena, rather than about our own code.